PNA.mdk

```diff
@@ -580,38 +580,16 @@ extern void send_to_port(PortId_t dest_port);
 
 The extern function `send_to_port` is used to direct a packet to a
 specified network port, or to a vport.  Invoking `send_to_port(x)` is
-supported only within the main control.  There are four cases to
+supported only within the main control.  There are two cases to
 consider, detailed below.
 
-+ current packet direction is `HOST_TO_NET`, and `x` is a network port id.
++ `x` is a network port id.
 
 Calling `send_to_port(x)` modifies hidden state for this packet, so
 that the packet will be transmitted out of the network port with id
-`x`, without being looped back.
-
-+ current packet direction is `NET_TO_HOST`, and `x` is a network port id.
-
-Calling `send_to_port(x)` modifies hidden state for this packet, so
-that when the packet is finished with the main control and main
-deparser, it will loop back in the host side, and later return to
-be processed by the main control in the HOST_TO_NET direction.  The
-hidden state will remain associated with the packet during that
-loopback, so that even if no further forwarding functions are
-called for the packet, it will be transmitted out of network port
```

Comment:
This deleted text and the next "hunk" deleted below eliminate some remaining mentions of NET_TO_HOST and HOST_TO_NET, and also simplify the description of `send_to_port` based upon the latest changes made elsewhere in the PNA spec.